blockchain/genesis/genesis.go

```diff
@@ -55,16 +55,16 @@ func defaultConfig() Genesis {
 			DardanellesBlockHeight:  1816201,
 			DaytonaBlockHeight:      3238921,
 			EasterBlockHeight:       4478761,
-			FbkMigrationBlockHeight: 4330441,
-			FairbankBlockHeight:     4339081,
+			FbkMigrationBlockHeight: 5159881,
```

Comment:
this can be even earlier. it doesn't hurt. maybe 10 hours earlier than fb height

Reply:
set to 12 hrs before